lib/mix/phoenix/schema.ex

```diff
@@ -175,6 +175,35 @@ defmodule Mix.Phoenix.Schema do
     |> Enum.into(%{}, fn {k, t} -> {k, type_to_default(k, t, action)} end)
   end
 
+  def value_as_map(%Date{} = date), do: %{day: date.day, month: date.month, year: date.year}
```

Comment:
@josevalim before adding docs & tests, what do you think about this solution? Since this change is required only for live tests, I tried to figure out a way to change these params only here.

Reply:
My first thought was to use this function:

https://github.com/phoenixframework/phoenix/blob/c8883af5582a38496e4b7e45e05d3a4d759a6caa/lib/mix/phoenix/schema.ex#L203

Then we could have those params created as we wish. But I couldn't figure out how 😢

Reply:
I like the current solution to be honest. My only suggestion is to call this function something like `live_form_value`. WDYT?